docs/conf.py

```diff
@@ -107,15 +107,15 @@
 # -- Options for HTMLHelp output ---------------------------------------------
 
 # Output file base name for HTML help builder.
-htmlhelp_basename = 'TungstenCIDashboard'
+htmlhelp_basename = 'ACID'
 
 
 # -- Options for manual page output ------------------------------------------
 
 # One entry per manual page. List of tuples
 # (source start file, name, description, authors, manual section).
 man_pages = [
-    (master_doc, 'TungstenCIDashboard', 'Tungsten CI dashboard Documentation',
+    (master_doc, 'ACIDCIDashboard', 'ACID CI dashboard Documentation',
```

Comment:
The `ACIDCIDashboard` is an ugly creature. Maybe use only `ACID`?

Reply:
And `ACID CI dashboard Documentation` also looks bad. Maybe just `ACID Documentation`.